src/battle_util.c

```diff
@@ -7210,6 +7210,24 @@ static u16 CalcMoveBasePower(u16 move, u8 battlerAtk, u8 battlerDef)
         if (gBattleMoves[gLastUsedMove].effect == EFFECT_FUSION_COMBO && move != gLastUsedMove)
             basePower *= 2;
         break;
+    case EFFECT_EXPANDING_FORCE:
+        if (gFieldStatuses & STATUS_FIELD_PSYCHIC_TERRAIN)
+            basePower = 120;
+        break;
+    case EFFECT_RISING_VOLTAGE:
+        #ifdef ITEM_EXPANSION   //Air Baloon exception with item expansion branch
+        if (gFieldStatuses & STATUS_FIELD_ELECTRIC_TERRAIN && !IS_BATTLER_OF_TYPE(battlerDef, TYPE_FLYING)
+            && gBattleMons[battlerDef].item != ITEM_AIR_BALLOON && gBattleMons[battlerDef].ability != ABILITY_LEVITATE)
+        #else
+        if (gFieldStatuses & STATUS_FIELD_ELECTRIC_TERRAIN && !IS_BATTLER_OF_TYPE(battlerDef, TYPE_FLYING)
+            && gBattleMons[battlerDef].ability != ABILITY_LEVITATE)
+        #endif
+        basePower *= 2;
```

Comment:
unsure if alignment matters from a compiler standpoint here, but the basePower *= 2 needs to be indented

Reply:
it doesn't, but it's good style to indent the part inside an if

Reply:
Instead of using a preproc directive here, which gets in the middle of readability, I think it would be better to define `ITEM_AIR_BALLOON` in `include/constants/battle_config.h`. There's other items already defined there anyway.

Reply:
It'd be better to use `IsBattlerGrounded(battlerDef)` here instead of checking for typing, item etc. This check also needs to account for Iron Ball, Gravity and a few other cases that are all covered by that function.